nixos/modules/services/home-automation/home-assistant.nix

```diff
@@ -517,7 +517,7 @@ in {
             description = ''
               List of ${domain}
               [blueprints](https://www.home-assistant.io/docs/blueprint/) to
-              install into {file}`''${configDir}/blueprints/${domain}`.
+              install into {file}`''${config.services.home-assistant.configDir}/blueprints/${domain}`.
```

Comment:
```suggestion
              install into {file}`''${services.home-assistant.configDir}/blueprints/${domain}`.
```

Reply:
or maybe `config` should be there, not sure

Reply:
I usually write it there, not sure if there is a consensus